docs/Installation.md

```diff
@@ -4,28 +4,84 @@ The Sumo Logic OT Distro can be run using either the binary file available in [G
 the container images stored in AWS Public ECR under the following repository:
 [public.ecr.aws/sumologic/sumologic-otel-collector](https://gallery.ecr.aws/sumologic/sumologic-otel-collector).
 
+- [Standalone](#standalone)
+  - [Linux on Intel](#linux-on-intel)
+  - [Linux on ARM](#linux-on-arm)
+  - [MacOS on Intel](#macos-on-intel)
+- [Container image](#container-image)
+
 ## Standalone
 
 The Sumo Logic OT Distro is a static Go binary.
 To run it as a standalone process you only need to run the binary file downloaded from
 [Github releases][github_releases] with an appropriate configuration.
 
-1. Set the release version variable:
+Follow the steps for your platform below.
+
+### Linux on Intel
```

Comment:
Maybe it's better to write "Linux on amd64"

Reply:
Yeah wasn't sure how to phrase it...

Reply:
I'd go with [X86-64 like Wikipedia redirects](https://en.wikipedia.org/wiki/X86-64).
Also to be fair I'd change the build names to that as well.

Reply:
Also it's the same that is shown in the Amazon ECR: 
![image](https://user-images.githubusercontent.com/58700054/129377298-0c0647bb-d7cd-4f50-b4f2-2d08a58a03e6.png)

Reply:
After discussing this in team, we decided to go with "amd64" and "arm64" for now as these are the names used by upstream and in binary names.